src/NDItk/MultigroupTable/src/read.hpp

```diff
@@ -15,18 +15,15 @@ void read( Iterator& iter, const Iterator& end ) {
 
       this->metadata_.read( keyword, iter, end );
     }
-    else if ( keyword == this->structure_.keyword() ) {
+    else if ( keyword.find( this->primary_structure_.keyword() ) == 0 ) {
```

Comment:
Why is this better than the previous way?

Reply:
this will be true for each keyword with e_bounds at the beginning of the string, meaning that it would pick up e_bounds_0 as well.